mpd_oled plugin 1.0.5

supercrab opened the pull request:
fixed missing config file

volumio commented:
IMHO this PR shall be closed and a new one needs to be created, since the node_modules are in the history and they will bloat the plugin repository

supercrab commented:
> v 1.0.5 breaks volumio!
> 
> ```
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: info: Loading plugin "mpd_oled"...
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: error: !!!! WARNING !!!!
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: error: The plugin user_interface/mpd_oled failed to load, setting it to stopped. Error: Error: Cannot find module 'kew'
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: Require stack:
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /data/plugins/user_interface/mpd_oled/index.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /volumio/app/pluginmanager.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /volumio/app/index.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /volumio/index.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: error: Stack trace: Error: Cannot find module 'kew'
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: Require stack:
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /data/plugins/user_interface/mpd_oled/index.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /volumio/app/pluginmanager.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /volumio/app/index.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: - /volumio/index.js
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Module.require (internal/modules/cjs/loader.js:952:19)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at require (internal/modules/cjs/helpers.js:88:18)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Object.<anonymous> (/data/plugins/user_interface/mpd_oled/index.js:6:12)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Module._compile (internal/modules/cjs/loader.js:1063:30)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Module.load (internal/modules/cjs/loader.js:928:32)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Module.require (internal/modules/cjs/loader.js:952:19)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at require (internal/modules/cjs/helpers.js:88:18)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at PluginManager.loadCorePlugin (/volumio/app/pluginmanager.js:235:29)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at /volumio/app/pluginmanager.js:338:34
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Array.forEach (<anonymous>)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at HashMap.<anonymous> (/volumio/app/pluginmanager.js:337:20)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at HashMap.forEach (/volumio/node_modules/hashmap/hashmap.js:171:10)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: error: !!!! WARNING !!!!
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: info: Loading i18n strings for locale fr
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: Updating browse sources language
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: info: CoreCommandRouter::executeOnPlugin: my_music , getDisabledSources
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: /volumio/app/index.js:1474
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:       if (typeof thisInterface.broadcastMessage === 'function') { return thisInterface.broadcastMessage(emit, payload); }
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:                                ^
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]: TypeError: Cannot read property 'broadcastMessage' of null
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at /volumio/app/index.js:1474:32
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at fastMap (/volumio/node_modules/fast.js/array/map.js:21:17)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Function.fastMap [as map] (/volumio/node_modules/fast.js/map.js:18:12)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at CoreCommandRouter.broadcastMessage (/volumio/app/index.js:1472:13)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at CoreMusicLibrary.pushBrowseSources (/volumio/app/musiclibrary.js:574:29)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at CoreMusicLibrary.updateBrowseSourcesLang (/volumio/app/musiclibrary.js:516:15)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at new CoreCommandRouter (/volumio/app/index.js:49:21)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Object.<anonymous> (/volumio/index.js:51:21)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Module._compile (internal/modules/cjs/loader.js:1063:30)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Module.load (internal/modules/cjs/loader.js:928:32)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
> Aug 02 10:02:33 volumio-rpi3 volumio[1286]:     at internal/main/run_main_module.js:17:47
> Aug 02 10:02:33 volumio-rpi3 systemd[1]: volumio.service: Main process exited, code=exited, status=1/FAILURE
> Aug 02 10:02:33 volumio-rpi3 systemd[1]: volumio.service: Failed with result 'exit-code'.
> ```
> 
> Please no node_modules! And I insist but category is 'system_hardware' because the plugin is related to a specific HW TY

- I have removed node_modules from my repository.
- I have changed the category to `system_hardware`.
- I'm unable to replicate the issue you are seeing.  I have removed and installed the plugin twice.  I haven't changed anything to do with dependencies
